services/core-api/app/api/now_applications/resources/now_application_list_resource.py

```diff
@@ -3,48 +3,53 @@
 from sqlalchemy_filters import apply_pagination, apply_sort
 from sqlalchemy import desc, func, or_
 
-
 from app.extensions import api
 from app.api.mines.mine.models.mine import Mine
 from app.api.mines.region.models.region import MineRegionCode
 from app.api.now_applications.models.notice_of_work_view import NoticeOfWorkView
+from app.api.now_applications.models.now_application_status import NOWApplicationStatus
 from app.api.now_applications.response_models import NOW_VIEW_LIST
 from app.api.utils.access_decorators import requires_role_view_all
-from app.api.utils.resources_mixins import UserMixin 
-
+from app.api.utils.resources_mixins import UserMixin
 
 PAGE_DEFAULT = 1
 PER_PAGE_DEFAULT = 25
+import sys
```

Comment:
Imports should be above const declarations

Reply:
My bad on forgetting to delete this. I simply brought it in to print to standard error output so I could display debug info in my log stream in my shell.